content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+# io.js Grupos de Trabajo
+
+Los Grupos de Trabajo (GT) de io.js son proyectos autónomos creados por el
+[Comité Tecnico (CT)](https://github.com/iojs/io.js/blob/v1.x/GOVERNANCE.md#technical-committee).
+
+Los Grupos de Trabajo pueden formarse en cualquier momento pero deben de ser ratificados por el CT.
+Una vez formados, el trabajo definido en la carta del Grupo de Trabajo es responsabilidad del Grupo de Trabajo en lugar del CT.
```

Comment:
* `Grupo de Trabajo` could be replaced by `GT` since the acronym was defined above.